staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -245,7 +245,7 @@ func (qs *queueSet) StartRequest(ctx context.Context, width uint, hashValue uint
 	// Step 0:
 	// Apply only concurrency limit, if zero queues desired
 	if qs.qCfg.DesiredNumQueues < 1 {
-		if qs.totSeatsInUse >= qs.dCfg.ConcurrencyLimit {
+		if !qs.canAccomodateSeatsLocked(int(width)) {
```

Comment:
nit: instead of converting to int, let's just make the param of canAccomodateSeatsLocked uint :)

Reply:
please ignore